plugins/engines/ospray/ispc/render/utils/AbstractRenderer.cpp

```diff
@@ -38,44 +38,47 @@ AbstractRenderer::AbstractRenderer()
 {
 }
 
+AbstractRenderer::~AbstractRenderer()
+{
+}
```

Comment:
note that you can also =default or omit the constructor / destructor if their implementation is empty